src/components/swapv2/LiquiditySourcesPanel/index.tsx

```diff
@@ -3,18 +3,28 @@ import { Flex, Box } from 'rebass'
 import { ArrowLeft } from 'react-feather'
 import styled from 'styled-components'
 import { t } from '@lingui/macro'
+import uniqBy from 'lodash/uniqBy'
```

Comment:
can you implement this by yourself. It's simple. 
The reason is this package isnt included in package.json, even it's existed in yarn.lock because of an other dependency use it, But I dont like implicit import like this

Reply:
Ok. I fixed it